src/docs/docsite.html.mustache

```diff
@@ -69,16 +69,30 @@
         <div class="site-toc">
           <ul>
             {{#site_toc}}
-              {{#link}}
-                <li class="toc-h{{depth}} toc-link {{#here}}toc-here{{/here}}">
-                  {{^here}}<a href="{{link}}">{{/here}}{{text}}{{^here}}</a>{{/here}}
+              {{#dropdown}}
+                <li class="toc-h{{depth}}">
+                  <a class="sidebar-nav-heading" class= "toc-drop" data-toggle="collapse" href="#{{id}}" aria-expanded="false" aria-controls="{{id}}">{{heading}}<span class="caret"></span></a>
```

Comment:
It'd be nice if the menu could be left open if the current page is below the collapsible menu item. I think that'd require plumbing a value for `aria-expanded` here from the python side.

It'd be nice if nested collapsible usages could work too, but that might be a little tricky.